tests/helpers/perf_run_helper.rb

```diff
@@ -683,10 +683,17 @@ def copy_system_logs(host)
                  "unknown_or_dev_job"
                end
 
-    archive_name = "#{job_name}__#{ENV['BUILD_ID']}__#{@gplt_timestamp}__system_logs.tgz"
     puppet_logdir = File.dirname on(host, puppet("config", "print", "logdir")).stdout.strip
-    archive_file_from(host, puppet_logdir,
-                      {}, @archive_root, archive_name)
+    logdir_root = puppet_logdir.split(File::SEPARATOR).select { |s| s.length >= 1 }.shift
```

Comment:
I had to run this through irb to find out what it does...

Reply:
I have added a comment to try and demystify it. :)

Reply:
it's better...  but I might only understand it because I now know what it does...
Maybe
# Get top-level dir name.  ex: /var/logs/puppet => var
That would be undeniably clear. :)

Reply:
This has been refactored out of existence.